integration/signing_test.go

```diff
@@ -113,5 +114,5 @@ func (s *SigningSuite) TestSignVerifySmoke(c *check.C) {
 	out, err = exec.Command(skopeoBinary, "standalone-verify", manifestPath,
 		dockerReference, s.fingerprint, sigOutput.Name()).CombinedOutput()
 	c.Assert(err, check.IsNil, check.Commentf("%s", out))
-	c.Assert(string(out), check.Equals, "Signature verified, digest "+fixtures.TestImageManifestDigest+"\n")
+	c.Assert(string(out), check.Equals, "Signature verified, digest "+testImageManifestDigest+"\n")
```

Comment:
… so _that’s_ why `fixtures` was a separate package. I have quite forgotten about this.

Wouldn’t it be cleaner to just copy&paste the relevant fixtures to the `integration` package then?